{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h1>Getting data files from NCBI SRA with eUtils</h1>\n",
    "<p>First, ensure that BioPython is installed.</p>\n",
    "<pre> sudo /home/.../anaconda3/bin/pip install biopython</pre>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 119,
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import Entrez \n",
    "import xmltodict\n",
    "import pandas as pd\n",
    "import numpy as np"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 120,
   "metadata": {},
   "outputs": [],
   "source": [
    "#to avoid anaonymous requests, adjust as needed\n",
    "Entrez.email = \"my@email.eu\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 137,
   "metadata": {},
   "outputs": [],
   "source": [
    "database=\"sra\"\n",
    "search_terms=\"estrogen receptor chip-seq AND homo sapiens[Organism]\"\n",
    "maxhits=200\n",
    "handle=Entrez.esearch(db=database,term=search_terms,retmax=maxhits)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 138,
   "metadata": {},
   "outputs": [],
   "source": [
    "records = Entrez.read(handle)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 139,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['5772180', '5772179', '5772178', '5772177', '5772176', '5679946', '5679945', '5679944', '5679943', '5679942', '5679941', '5679940', '5679939', '5679938', '5679937', '5026291', '5026290', '5026289', '5026288', '5026287', '5026286', '5026285', '5026284', '5026283', '5026282', '5026281', '4463046', '4463045', '4463044', '4463043', '4463042', '4463041', '4463040', '4463039', '4463038', '4463037', '4463036', '4463035', '4463034', '4463033', '4463032', '4463031', '4463030', '4463029', '4417855', '4417854', '4417853', '4417852', '4417851', '4417850', '4417849', '4417848', '4417847', '4417846', '4417845', '4417844', '4417843', '4417842', '4417841', '4417840', '4417839', '4417838', '4417837', '4417836', '4417835', '4417834', '4417833', '4417832', '4285995', '4285994', '4285993', '4285992', '4285991', '4285990', '4285989', '4285988', '4285987', '4285986', '4175998', '4175997', '4175996', '4175995', '4175994', '4175993', '4175992', '4175991', '4175990', '4175989', '4175988', '4175987', '4148037', '4148036', '4148035', '4148034', '4148033', '4148032', '4148031', '4148030', '4148029', '4148028', '4148027', '4148026', '4148025', '4148024', '4148023', '4148022', '4148021', '4148020', '4148019', '4148018', '4148017', '4148016', '4148015', '4148014', '4148013', '4148012', '4148011', '4148010', '4148009', '4148008', '4148007', '4148006', '4148005', '4148004', '4148003', '4148002', '4148001', '4148000', '4147999', '4147998', '4147997', '4147996', '4147995', '4147994', '4147993', '4147992', '4147991', '4147990', '4147989', '4162881', '4162880', '4162879', '4162878', '4162877', '4162876', '4162875', '4162874', '4162873', '4162872', '4162871', '4162870', '4162868', '4162866', '4162864', '4162862', '4162860', '4162858', '4162856', '4162854', '4162852', '4162850', '4162848', '4152748', '4152747', '4152746', '4152745', '4152744', '4152743', '4152742', '4152741', '4152740', '4152739', '4152738', '4152737', '4152736', '4152735', '4152734', '4124032', '4124031', '4124030', '4124029', '4124028', '4124027', '4124026', '4124025', '4124024', '4124023', '4124022', '3676536', '3676535', '3676534', '3676533', '3676532', '3676531', '3676530', '3676529', '3676528', '3676527', '3676526', '3676525']"
      ]
     },
     "execution_count": 139,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "identifiers = records['IdList']\n",
    "identifiers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 140,
   "metadata": {},
   "outputs": [],
   "source": [
    "handle = Entrez.efetch(db=database, id=identifiers, retmax=\"200\", retmode=\"text\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 141,
   "metadata": {},
   "outputs": [],
   "source": [
    "#ecord = Entrez.read(handle,validate=False\n",
    "#records=Entrez.parse(handle)\n",
    "#for record in records:\n",
    "#rint(handle.read())\n",
    "text=handle.read()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 142,
   "metadata": {},
   "outputs": [],
   "source": [
    "import xmltodict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "metadata": {},
   "outputs": [],
   "source": [
    "doc=xmltodict.parse(text)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h2>Parsing the output of SRA/eUtils</h2>\n",
    "<p>The <tt>Entrez.efetch</tt> command returns XML, which the <tt>xmltodict</tt> package transforms into a hierarchical \n",
    "Python dictionary. The top element is <tt>EXPERIMENT_PACKAGE_SET</tt>. Underneath that there is a list of \n",
    "<tt>EXPERIMENT_PACKAGE</tt> objects. There should be one such <tt>EXPERIMENT_PACKAGE</tt> for each ChIP-seq experiment that SRA returns.</p>\n",
    "<p>The following function extracts information from an individual <tt>EXPERIMENT_PACKAGE</tt> and puts the info\n",
    "    into a row of a pandas dataframe.</p>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "def parse_experiment_package(ep,i,df):\n",
    "    #experiment=ep['EXPERIMENT']\n",
    "    for item in ep:\n",
    "        if (item=='EXPERIMENT'):\n",
    "            experiment=ep[item]\n",
    "            experiment_alias = experiment['@alias']\n",
    "            experiment_accession= experiment['@accession']\n",
    "            title=experiment['TITLE']\n",
    "            studyref=experiment['STUDY_REF'] # this is an ordered dict\n",
    "            study_accession=studyref['@accession']\n",
    "            study_refname=studyref['@refname']\n",
    "            #print(study_accession,study_refname)\n",
    "            design=experiment['DESIGN']# this is an ordered dict\n",
    "            sample_descriptor=design['SAMPLE_DESCRIPTOR']\n",
    "            sample_accession=sample_descriptor['@accession']\n",
    "            library_descriptor=design['LIBRARY_DESCRIPTOR']# this is an ordered dict\n",
    "            library_strategy=library_descriptor['LIBRARY_STRATEGY']\n",
    "            library_construction=library_descriptor['LIBRARY_CONSTRUCTION_PROTOCOL']\n",
    "            #print(sample_accession,library_strategy,library_construction)\n",
    "            platform=experiment['PLATFORM']\n",
    "            illumina=platform['ILLUMINA']\n",
    "            instrument=illumina['INSTRUMENT_MODEL']\n",
    "            #print(instrument)\n",
    "            #experiment_links=experiment['EXPERIMENT_LINKS']\n",
    "            #experiment_attributes=experiment['EXPERIMENT_ATTRIBUTES']\n",
    "            submission=ep['SUBMISSION']\n",
    "            submission_alias=submission['@alias']\n",
    "            submission_accession=submission['@accession']\n",
    "            study=ep['STUDY']\n",
    "            descriptor=study['DESCRIPTOR']\n",
    "            study_title=descriptor['STUDY_TITLE']\n",
    "            study_type=descriptor['STUDY_TYPE']\n",
    "            study_abstract=descriptor['STUDY_ABSTRACT']\n",
    "            df.loc[i]=[experiment_alias,experiment_accession,title,\n",
    "               study_accession,study_refname,sample_accession,library_strategy,library_construction,\n",
    "               instrument,submission_alias,submission_accession,study_title,study_type,study_abstract]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<h2>Create a pandas dataframe to hold the results</h2>\n",
    "<p>\n",
    "We will now create a pandas dataframe to hold the results, defining columns that match those sought after in the above function.\n",
    "Not good style, refactor later on!</p>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 145,
   "metadata": {},
   "outputs": [],
   "source": [
    "numberOfRows=maxhits\n",
    "df=pd.DataFrame(index=np.arange(0,numberOfRows),columns=('experiment_alias','experiment_accession','title',\n",
    "               'study_accession','study_refname','sample_accession','library_strategy','library_construction',\n",
    "               'instrument','submission_alias','submission_accession','study_title','study_type','study_abstract'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 147,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n",
      "Could not parse this entry\n"
     ]
    }
   ],
   "source": [
    "\n",
    "doc2=doc['EXPERIMENT_PACKAGE_SET']\n",
    "#the following gets a list of EXPERIMENT_PACKAGE objects\n",
    "doc3=doc2['EXPERIMENT_PACKAGE']\n",
    "i=0\n",
    "for ep in doc3:\n",
    "    try:\n",
    "        parse_experiment_package (ep,i,df)\n",
    "        i=i+1\n",
    "    except:\n",
    "        print(\"Could not parse this entry\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>experiment_alias</th>\n",
       "      <th>experiment_accession</th>\n",
       "      <th>title</th>\n",
       "      <th>study_accession</th>\n",
       "      <th>study_refname</th>\n",
       "      <th>sample_accession</th>\n",
       "      <th>library_strategy</th>\n",
       "      <th>library_construction</th>\n",
       "      <th>instrument</th>\n",
       "      <th>submission_alias</th>\n",
       "      <th>submission_accession</th>\n",
       "      <th>study_title</th>\n",
       "      <th>study_type</th>\n",
       "      <th>study_abstract</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSM3212403</td>\n",
       "      <td>SRX4284795</td>\n",
       "      <td>GSM3212403: T47D_YFP_noE2_HA_ChIPseq; Homo sap...</td>\n",
       "      <td>SRP151140</td>\n",
       "      <td>GSE116170</td>\n",
       "      <td>SRS3448952</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were grown hormone-deprived media for 7 ...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE116170</td>\n",
       "      <td>SRA726535</td>\n",
       "      <td>Transcriptional properties of estrogen recepto...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>RNA sequencing (RNA-seq) detects estrogen rece...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSM3212402</td>\n",
       "      <td>SRX4284794</td>\n",
       "      <td>GSM3212402: T47D_ESR1NOP2_noE2_HA_ChIPseq; Hom...</td>\n",
       "      <td>SRP151140</td>\n",
       "      <td>GSE116170</td>\n",
       "      <td>SRS3448951</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were grown hormone-deprived media for 7 ...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE116170</td>\n",
       "      <td>SRA726535</td>\n",
       "      <td>Transcriptional properties of estrogen recepto...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>RNA sequencing (RNA-seq) detects estrogen rece...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSM3212401</td>\n",
       "      <td>SRX4284793</td>\n",
       "      <td>GSM3212401: T47D_ESR1PCDH11X_noE2_HA_ChIPseq; ...</td>\n",
       "      <td>SRP151140</td>\n",
       "      <td>GSE116170</td>\n",
       "      <td>SRS3448950</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were grown hormone-deprived media for 7 ...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE116170</td>\n",
       "      <td>SRA726535</td>\n",
       "      <td>Transcriptional properties of estrogen recepto...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>RNA sequencing (RNA-seq) detects estrogen rece...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSM3212400</td>\n",
       "      <td>SRX4284792</td>\n",
       "      <td>GSM3212400: T47D_ESR1YAP1_noE2_HA_ChIPseq; Hom...</td>\n",
       "      <td>SRP151140</td>\n",
       "      <td>GSE116170</td>\n",
       "      <td>SRS3448949</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were grown hormone-deprived media for 7 ...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE116170</td>\n",
       "      <td>SRA726535</td>\n",
       "      <td>Transcriptional properties of estrogen recepto...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>RNA sequencing (RNA-seq) detects estrogen rece...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSM3212399</td>\n",
       "      <td>SRX4284791</td>\n",
       "      <td>GSM3212399: T47D_ESR1WT_noE2_HA_ChIPseq; Homo ...</td>\n",
       "      <td>SRP151140</td>\n",
       "      <td>GSE116170</td>\n",
       "      <td>SRS3448967</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were grown hormone-deprived media for 7 ...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE116170</td>\n",
       "      <td>SRA726535</td>\n",
       "      <td>Transcriptional properties of estrogen recepto...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>RNA sequencing (RNA-seq) detects estrogen rece...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>GSM3184880</td>\n",
       "      <td>SRX4193150</td>\n",
       "      <td>GSM3184880: ERalpha s10 E2 ChIPSeq; Homo sapie...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403984</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>GSM3184879</td>\n",
       "      <td>SRX4193149</td>\n",
       "      <td>GSM3184879: ERalpha s9 Fulv ChIPSeq; Homo sapi...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403986</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>GSM3184878</td>\n",
       "      <td>SRX4193148</td>\n",
       "      <td>GSM3184878: ERalpha s8 Tam ChIPSeq; Homo sapie...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403983</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>GSM3184877</td>\n",
       "      <td>SRX4193147</td>\n",
       "      <td>GSM3184877: ERalpha s7 s5942 ChIPSeq; Homo sap...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403982</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>GSM3184876</td>\n",
       "      <td>SRX4193146</td>\n",
       "      <td>GSM3184876: ERalpha s6 5942 ChIPSeq; Homo sapi...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403979</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>GSM3184875</td>\n",
       "      <td>SRX4193145</td>\n",
       "      <td>GSM3184875: ERalpha s5 RAD1901 ChIPSeq; Homo s...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403980</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>GSM3184874</td>\n",
       "      <td>SRX4193144</td>\n",
       "      <td>GSM3184874: ERalpha s4 800 ChIPSeq; Homo sapie...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403978</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>GSM3184873</td>\n",
       "      <td>SRX4193143</td>\n",
       "      <td>GSM3184873: ERalpha s3 s6545 ChIPSeq; Homo sap...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403976</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>GSM3184872</td>\n",
       "      <td>SRX4193142</td>\n",
       "      <td>GSM3184872: ERalpha s2 6545 ChIPSeq; Homo sapi...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403977</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>GSM3184871</td>\n",
       "      <td>SRX4193141</td>\n",
       "      <td>GSM3184871: ERalpha s1 DMSO ChIPSeq; Homo sapi...</td>\n",
       "      <td>SRP150236</td>\n",
       "      <td>GSE115607</td>\n",
       "      <td>SRS3403981</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were formaldehyde fixed for 15 minutes (...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE115607</td>\n",
       "      <td>SRA719221</td>\n",
       "      <td>ChIP-Seq analysis of estrogen deprived MCF7 ce...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>The goal of this experiment was to interrogate...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>GSM2973404</td>\n",
       "      <td>SRX3630824</td>\n",
       "      <td>GSM2973404: H3K27ac_input_8hr; Homo sapiens; C...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897537</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>GSM2973403</td>\n",
       "      <td>SRX3630823</td>\n",
       "      <td>GSM2973403: H3K27ac_DMSO_8hr; Homo sapiens; Ch...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897536</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>GSM2973402</td>\n",
       "      <td>SRX3630822</td>\n",
       "      <td>GSM2973402: H3K27ac_Dex_8hr; Homo sapiens; ChI...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897534</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>GSM2973401</td>\n",
       "      <td>SRX3630821</td>\n",
       "      <td>GSM2973401: H3K27ac_E2_Dex_8hr; Homo sapiens; ...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897535</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>GSM2973400</td>\n",
       "      <td>SRX3630820</td>\n",
       "      <td>GSM2973400: H3K27ac_E2_8hr; Homo sapiens; ChIP...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897533</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>GSM2973399</td>\n",
       "      <td>SRX3630819</td>\n",
       "      <td>GSM2973399: Ishi_DMSO_ChIP_GR; Homo sapiens; C...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897532</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>GSM2973398</td>\n",
       "      <td>SRX3630818</td>\n",
       "      <td>GSM2973398: Ishi_DMSO_ChIP_ER; Homo sapiens; C...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897530</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>GSM2973397</td>\n",
       "      <td>SRX3630817</td>\n",
       "      <td>GSM2973397: Ishi_Dex_ChIP_GR; Homo sapiens; Ch...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897531</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>GSM2973396</td>\n",
       "      <td>SRX3630816</td>\n",
       "      <td>GSM2973396: Ishi_E2_ChIP_ER; Homo sapiens; ChI...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897527</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>GSM2973395</td>\n",
       "      <td>SRX3630815</td>\n",
       "      <td>GSM2973395: Ishi_Dex_E2_ChIP_GR; Homo sapiens;...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897529</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>GSM2973394</td>\n",
       "      <td>SRX3630814</td>\n",
       "      <td>GSM2973394: Ishi_Dex_E2_ChIP_ER; Homo sapiens;...</td>\n",
       "      <td>SRP131799</td>\n",
       "      <td>GSE109891</td>\n",
       "      <td>SRS2897528</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were fixed in 1% formaldehyde for 10 min...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE109891</td>\n",
       "      <td>SRA653941</td>\n",
       "      <td>Clinical and genomic crosstalk between glucoco...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>Steroid hormone receptors are simultaneously a...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>GSM2752601</td>\n",
       "      <td>SRX3123833</td>\n",
       "      <td>GSM2752601: jc2929_MCF7_Input_AZD2014; Homo sa...</td>\n",
       "      <td>SRP116074</td>\n",
       "      <td>GSE103023</td>\n",
       "      <td>SRS2458192</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were crosslinked for 10min using 1% form...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE103023</td>\n",
       "      <td>SRA602384</td>\n",
       "      <td>Estrogen receptor ChIP-seq in response to mTOR...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We have used ChIP-seq to profile binding of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>GSM2752600</td>\n",
       "      <td>SRX3123832</td>\n",
       "      <td>GSM2752600: jc2928_MCF7_Input_RAD001; Homo sap...</td>\n",
       "      <td>SRP116074</td>\n",
       "      <td>GSE103023</td>\n",
       "      <td>SRS2458191</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were crosslinked for 10min using 1% form...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE103023</td>\n",
       "      <td>SRA602384</td>\n",
       "      <td>Estrogen receptor ChIP-seq in response to mTOR...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We have used ChIP-seq to profile binding of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>GSM2752599</td>\n",
       "      <td>SRX3123831</td>\n",
       "      <td>GSM2752599: jc2927_MCF7_Input_Veh; Homo sapien...</td>\n",
       "      <td>SRP116074</td>\n",
       "      <td>GSE103023</td>\n",
       "      <td>SRS2458190</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were crosslinked for 10min using 1% form...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE103023</td>\n",
       "      <td>SRA602384</td>\n",
       "      <td>Estrogen receptor ChIP-seq in response to mTOR...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We have used ChIP-seq to profile binding of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>GSM2752598</td>\n",
       "      <td>SRX3123830</td>\n",
       "      <td>GSM2752598: jc2926_MCF7_ER_AZD2014; Homo sapie...</td>\n",
       "      <td>SRP116074</td>\n",
       "      <td>GSE103023</td>\n",
       "      <td>SRS2458189</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cells were crosslinked for 10min using 1% form...</td>\n",
       "      <td>Illumina HiSeq 2500</td>\n",
       "      <td>GEO: GSE103023</td>\n",
       "      <td>SRA602384</td>\n",
       "      <td>Estrogen receptor ChIP-seq in response to mTOR...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We have used ChIP-seq to profile binding of th...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>GSM2476831</td>\n",
       "      <td>SRX2539936</td>\n",
       "      <td>GSM2476831: T47D-Y537S-DOXE2-TS8-MP2496; Homo ...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959644</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>GSM2476830</td>\n",
       "      <td>SRX2539935</td>\n",
       "      <td>GSM2476830: T47D-Y537S-DOX-TS6-MP2496; Homo sa...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959642</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>GSM2476829</td>\n",
       "      <td>SRX2539934</td>\n",
       "      <td>GSM2476829: T47D-Y537S-E2-TS7-MP2496; Homo sap...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959643</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>GSM2476828</td>\n",
       "      <td>SRX2539933</td>\n",
       "      <td>GSM2476828: Y537S-DOXE2-rep2-S2-8-MP2867; Homo...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959641</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>GSM2476827</td>\n",
       "      <td>SRX2539932</td>\n",
       "      <td>GSM2476827: Y537S-DOXE2-rep1-S2-7-MP2867; Homo...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959640</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>GSM2476826</td>\n",
       "      <td>SRX2539931</td>\n",
       "      <td>GSM2476826: Y537S-DOXE2-SSS5-MP2498; Homo sapi...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959639</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>GSM2476825</td>\n",
       "      <td>SRX2539930</td>\n",
       "      <td>GSM2476825: Y537S-DOX-rep2-S2-4-MP2867; Homo s...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959638</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>GSM2476824</td>\n",
       "      <td>SRX2539929</td>\n",
       "      <td>GSM2476824: Y537S-DOX-rep1-S2-3-MP2867; Homo s...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959636</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>GSM2476823</td>\n",
       "      <td>SRX2539928</td>\n",
       "      <td>GSM2476823: Y537S-DOX-SSS3-MP2498; Homo sapien...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959637</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>GSM2476822</td>\n",
       "      <td>SRX2539927</td>\n",
       "      <td>GSM2476822: Y537S-E2-rep2-S2-6-MP2867; Homo sa...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959635</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>GSM2476821</td>\n",
       "      <td>SRX2539926</td>\n",
       "      <td>GSM2476821: Y537S-E2-rep1-S2-5-MP2867; Homo sa...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959633</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>GSM2476820</td>\n",
       "      <td>SRX2539925</td>\n",
       "      <td>GSM2476820: Y537S-E2-SSS4-MP2498; Homo sapiens...</td>\n",
       "      <td>SRP098916</td>\n",
       "      <td>GSE94493</td>\n",
       "      <td>SRS1959634</td>\n",
       "      <td>ChIP-Seq</td>\n",
       "      <td>Cell lines before ChIP were then either introd...</td>\n",
       "      <td>NextSeq 500</td>\n",
       "      <td>GEO: GSE94493</td>\n",
       "      <td>SRA535839</td>\n",
       "      <td>ChIPseq and RNAseq of ER mutant breast cancer ...</td>\n",
       "      <td>{'@existing_study_type': 'Other'}</td>\n",
       "      <td>We performed several extensive phenotypic, gen...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>200 rows × 14 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    experiment_alias experiment_accession  \\\n",
       "0         GSM3212403           SRX4284795   \n",
       "1         GSM3212402           SRX4284794   \n",
       "2         GSM3212401           SRX4284793   \n",
       "3         GSM3212400           SRX4284792   \n",
       "4         GSM3212399           SRX4284791   \n",
       "5         GSM3184880           SRX4193150   \n",
       "6         GSM3184879           SRX4193149   \n",
       "7         GSM3184878           SRX4193148   \n",
       "8         GSM3184877           SRX4193147   \n",
       "9         GSM3184876           SRX4193146   \n",
       "10        GSM3184875           SRX4193145   \n",
       "11        GSM3184874           SRX4193144   \n",
       "12        GSM3184873           SRX4193143   \n",
       "13        GSM3184872           SRX4193142   \n",
       "14        GSM3184871           SRX4193141   \n",
       "15        GSM2973404           SRX3630824   \n",
       "16        GSM2973403           SRX3630823   \n",
       "17        GSM2973402           SRX3630822   \n",
       "18        GSM2973401           SRX3630821   \n",
       "19        GSM2973400           SRX3630820   \n",
       "20        GSM2973399           SRX3630819   \n",
       "21        GSM2973398           SRX3630818   \n",
       "22        GSM2973397           SRX3630817   \n",
       "23        GSM2973396           SRX3630816   \n",
       "24        GSM2973395           SRX3630815   \n",
       "25        GSM2973394           SRX3630814   \n",
       "26        GSM2752601           SRX3123833   \n",
       "27        GSM2752600           SRX3123832   \n",
       "28        GSM2752599           SRX3123831   \n",
       "29        GSM2752598           SRX3123830   \n",
       "..               ...                  ...   \n",
       "170       GSM2476831           SRX2539936   \n",
       "171       GSM2476830           SRX2539935   \n",
       "172       GSM2476829           SRX2539934   \n",
       "173       GSM2476828           SRX2539933   \n",
       "174       GSM2476827           SRX2539932   \n",
       "175       GSM2476826           SRX2539931   \n",
       "176       GSM2476825           SRX2539930   \n",
       "177       GSM2476824           SRX2539929   \n",
       "178       GSM2476823           SRX2539928   \n",
       "179       GSM2476822           SRX2539927   \n",
       "180       GSM2476821           SRX2539926   \n",
       "181       GSM2476820           SRX2539925   \n",
       "182              NaN                  NaN   \n",
       "183              NaN                  NaN   \n",
       "184              NaN                  NaN   \n",
       "185              NaN                  NaN   \n",
       "186              NaN                  NaN   \n",
       "187              NaN                  NaN   \n",
       "188              NaN                  NaN   \n",
       "189              NaN                  NaN   \n",
       "190              NaN                  NaN   \n",
       "191              NaN                  NaN   \n",
       "192              NaN                  NaN   \n",
       "193              NaN                  NaN   \n",
       "194              NaN                  NaN   \n",
       "195              NaN                  NaN   \n",
       "196              NaN                  NaN   \n",
       "197              NaN                  NaN   \n",
       "198              NaN                  NaN   \n",
       "199              NaN                  NaN   \n",
       "\n",
       "                                                 title study_accession  \\\n",
       "0    GSM3212403: T47D_YFP_noE2_HA_ChIPseq; Homo sap...       SRP151140   \n",
       "1    GSM3212402: T47D_ESR1NOP2_noE2_HA_ChIPseq; Hom...       SRP151140   \n",
       "2    GSM3212401: T47D_ESR1PCDH11X_noE2_HA_ChIPseq; ...       SRP151140   \n",
       "3    GSM3212400: T47D_ESR1YAP1_noE2_HA_ChIPseq; Hom...       SRP151140   \n",
       "4    GSM3212399: T47D_ESR1WT_noE2_HA_ChIPseq; Homo ...       SRP151140   \n",
       "5    GSM3184880: ERalpha s10 E2 ChIPSeq; Homo sapie...       SRP150236   \n",
       "6    GSM3184879: ERalpha s9 Fulv ChIPSeq; Homo sapi...       SRP150236   \n",
       "7    GSM3184878: ERalpha s8 Tam ChIPSeq; Homo sapie...       SRP150236   \n",
       "8    GSM3184877: ERalpha s7 s5942 ChIPSeq; Homo sap...       SRP150236   \n",
       "9    GSM3184876: ERalpha s6 5942 ChIPSeq; Homo sapi...       SRP150236   \n",
       "10   GSM3184875: ERalpha s5 RAD1901 ChIPSeq; Homo s...       SRP150236   \n",
       "11   GSM3184874: ERalpha s4 800 ChIPSeq; Homo sapie...       SRP150236   \n",
       "12   GSM3184873: ERalpha s3 s6545 ChIPSeq; Homo sap...       SRP150236   \n",
       "13   GSM3184872: ERalpha s2 6545 ChIPSeq; Homo sapi...       SRP150236   \n",
       "14   GSM3184871: ERalpha s1 DMSO ChIPSeq; Homo sapi...       SRP150236   \n",
       "15   GSM2973404: H3K27ac_input_8hr; Homo sapiens; C...       SRP131799   \n",
       "16   GSM2973403: H3K27ac_DMSO_8hr; Homo sapiens; Ch...       SRP131799   \n",
       "17   GSM2973402: H3K27ac_Dex_8hr; Homo sapiens; ChI...       SRP131799   \n",
       "18   GSM2973401: H3K27ac_E2_Dex_8hr; Homo sapiens; ...       SRP131799   \n",
       "19   GSM2973400: H3K27ac_E2_8hr; Homo sapiens; ChIP...       SRP131799   \n",
       "20   GSM2973399: Ishi_DMSO_ChIP_GR; Homo sapiens; C...       SRP131799   \n",
       "21   GSM2973398: Ishi_DMSO_ChIP_ER; Homo sapiens; C...       SRP131799   \n",
       "22   GSM2973397: Ishi_Dex_ChIP_GR; Homo sapiens; Ch...       SRP131799   \n",
       "23   GSM2973396: Ishi_E2_ChIP_ER; Homo sapiens; ChI...       SRP131799   \n",
       "24   GSM2973395: Ishi_Dex_E2_ChIP_GR; Homo sapiens;...       SRP131799   \n",
       "25   GSM2973394: Ishi_Dex_E2_ChIP_ER; Homo sapiens;...       SRP131799   \n",
       "26   GSM2752601: jc2929_MCF7_Input_AZD2014; Homo sa...       SRP116074   \n",
       "27   GSM2752600: jc2928_MCF7_Input_RAD001; Homo sap...       SRP116074   \n",
       "28   GSM2752599: jc2927_MCF7_Input_Veh; Homo sapien...       SRP116074   \n",
       "29   GSM2752598: jc2926_MCF7_ER_AZD2014; Homo sapie...       SRP116074   \n",
       "..                                                 ...             ...   \n",
       "170  GSM2476831: T47D-Y537S-DOXE2-TS8-MP2496; Homo ...       SRP098916   \n",
       "171  GSM2476830: T47D-Y537S-DOX-TS6-MP2496; Homo sa...       SRP098916   \n",
       "172  GSM2476829: T47D-Y537S-E2-TS7-MP2496; Homo sap...       SRP098916   \n",
       "173  GSM2476828: Y537S-DOXE2-rep2-S2-8-MP2867; Homo...       SRP098916   \n",
       "174  GSM2476827: Y537S-DOXE2-rep1-S2-7-MP2867; Homo...       SRP098916   \n",
       "175  GSM2476826: Y537S-DOXE2-SSS5-MP2498; Homo sapi...       SRP098916   \n",
       "176  GSM2476825: Y537S-DOX-rep2-S2-4-MP2867; Homo s...       SRP098916   \n",
       "177  GSM2476824: Y537S-DOX-rep1-S2-3-MP2867; Homo s...       SRP098916   \n",
       "178  GSM2476823: Y537S-DOX-SSS3-MP2498; Homo sapien...       SRP098916   \n",
       "179  GSM2476822: Y537S-E2-rep2-S2-6-MP2867; Homo sa...       SRP098916   \n",
       "180  GSM2476821: Y537S-E2-rep1-S2-5-MP2867; Homo sa...       SRP098916   \n",
       "181  GSM2476820: Y537S-E2-SSS4-MP2498; Homo sapiens...       SRP098916   \n",
       "182                                                NaN             NaN   \n",
       "183                                                NaN             NaN   \n",
       "184                                                NaN             NaN   \n",
       "185                                                NaN             NaN   \n",
       "186                                                NaN             NaN   \n",
       "187                                                NaN             NaN   \n",
       "188                                                NaN             NaN   \n",
       "189                                                NaN             NaN   \n",
       "190                                                NaN             NaN   \n",
       "191                                                NaN             NaN   \n",
       "192                                                NaN             NaN   \n",
       "193                                                NaN             NaN   \n",
       "194                                                NaN             NaN   \n",
       "195                                                NaN             NaN   \n",
       "196                                                NaN             NaN   \n",
       "197                                                NaN             NaN   \n",
       "198                                                NaN             NaN   \n",
       "199                                                NaN             NaN   \n",
       "\n",
       "    study_refname sample_accession library_strategy  \\\n",
       "0       GSE116170       SRS3448952         ChIP-Seq   \n",
       "1       GSE116170       SRS3448951         ChIP-Seq   \n",
       "2       GSE116170       SRS3448950         ChIP-Seq   \n",
       "3       GSE116170       SRS3448949         ChIP-Seq   \n",
       "4       GSE116170       SRS3448967         ChIP-Seq   \n",
       "5       GSE115607       SRS3403984         ChIP-Seq   \n",
       "6       GSE115607       SRS3403986         ChIP-Seq   \n",
       "7       GSE115607       SRS3403983         ChIP-Seq   \n",
       "8       GSE115607       SRS3403982         ChIP-Seq   \n",
       "9       GSE115607       SRS3403979         ChIP-Seq   \n",
       "10      GSE115607       SRS3403980         ChIP-Seq   \n",
       "11      GSE115607       SRS3403978         ChIP-Seq   \n",
       "12      GSE115607       SRS3403976         ChIP-Seq   \n",
       "13      GSE115607       SRS3403977         ChIP-Seq   \n",
       "14      GSE115607       SRS3403981         ChIP-Seq   \n",
       "15      GSE109891       SRS2897537         ChIP-Seq   \n",
       "16      GSE109891       SRS2897536         ChIP-Seq   \n",
       "17      GSE109891       SRS2897534         ChIP-Seq   \n",
       "18      GSE109891       SRS2897535         ChIP-Seq   \n",
       "19      GSE109891       SRS2897533         ChIP-Seq   \n",
       "20      GSE109891       SRS2897532         ChIP-Seq   \n",
       "21      GSE109891       SRS2897530         ChIP-Seq   \n",
       "22      GSE109891       SRS2897531         ChIP-Seq   \n",
       "23      GSE109891       SRS2897527         ChIP-Seq   \n",
       "24      GSE109891       SRS2897529         ChIP-Seq   \n",
       "25      GSE109891       SRS2897528         ChIP-Seq   \n",
       "26      GSE103023       SRS2458192         ChIP-Seq   \n",
       "27      GSE103023       SRS2458191         ChIP-Seq   \n",
       "28      GSE103023       SRS2458190         ChIP-Seq   \n",
       "29      GSE103023       SRS2458189         ChIP-Seq   \n",
       "..            ...              ...              ...   \n",
       "170      GSE94493       SRS1959644         ChIP-Seq   \n",
       "171      GSE94493       SRS1959642         ChIP-Seq   \n",
       "172      GSE94493       SRS1959643         ChIP-Seq   \n",
       "173      GSE94493       SRS1959641         ChIP-Seq   \n",
       "174      GSE94493       SRS1959640         ChIP-Seq   \n",
       "175      GSE94493       SRS1959639         ChIP-Seq   \n",
       "176      GSE94493       SRS1959638         ChIP-Seq   \n",
       "177      GSE94493       SRS1959636         ChIP-Seq   \n",
       "178      GSE94493       SRS1959637         ChIP-Seq   \n",
       "179      GSE94493       SRS1959635         ChIP-Seq   \n",
       "180      GSE94493       SRS1959633         ChIP-Seq   \n",
       "181      GSE94493       SRS1959634         ChIP-Seq   \n",
       "182           NaN              NaN              NaN   \n",
       "183           NaN              NaN              NaN   \n",
       "184           NaN              NaN              NaN   \n",
       "185           NaN              NaN              NaN   \n",
       "186           NaN              NaN              NaN   \n",
       "187           NaN              NaN              NaN   \n",
       "188           NaN              NaN              NaN   \n",
       "189           NaN              NaN              NaN   \n",
       "190           NaN              NaN              NaN   \n",
       "191           NaN              NaN              NaN   \n",
       "192           NaN              NaN              NaN   \n",
       "193           NaN              NaN              NaN   \n",
       "194           NaN              NaN              NaN   \n",
       "195           NaN              NaN              NaN   \n",
       "196           NaN              NaN              NaN   \n",
       "197           NaN              NaN              NaN   \n",
       "198           NaN              NaN              NaN   \n",
       "199           NaN              NaN              NaN   \n",
       "\n",
       "                                  library_construction           instrument  \\\n",
       "0    Cells were grown hormone-deprived media for 7 ...  Illumina HiSeq 2500   \n",
       "1    Cells were grown hormone-deprived media for 7 ...  Illumina HiSeq 2500   \n",
       "2    Cells were grown hormone-deprived media for 7 ...  Illumina HiSeq 2500   \n",
       "3    Cells were grown hormone-deprived media for 7 ...  Illumina HiSeq 2500   \n",
       "4    Cells were grown hormone-deprived media for 7 ...  Illumina HiSeq 2500   \n",
       "5    Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "6    Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "7    Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "8    Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "9    Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "10   Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "11   Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "12   Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "13   Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "14   Cells were formaldehyde fixed for 15 minutes (...          NextSeq 500   \n",
       "15   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "16   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "17   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "18   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "19   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "20   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "21   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "22   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "23   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "24   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "25   Cells were fixed in 1% formaldehyde for 10 min...  Illumina HiSeq 2500   \n",
       "26   Cells were crosslinked for 10min using 1% form...  Illumina HiSeq 2500   \n",
       "27   Cells were crosslinked for 10min using 1% form...  Illumina HiSeq 2500   \n",
       "28   Cells were crosslinked for 10min using 1% form...  Illumina HiSeq 2500   \n",
       "29   Cells were crosslinked for 10min using 1% form...  Illumina HiSeq 2500   \n",
       "..                                                 ...                  ...   \n",
       "170  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "171  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "172  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "173  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "174  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "175  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "176  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "177  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "178  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "179  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "180  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "181  Cell lines before ChIP were then either introd...          NextSeq 500   \n",
       "182                                                NaN                  NaN   \n",
       "183                                                NaN                  NaN   \n",
       "184                                                NaN                  NaN   \n",
       "185                                                NaN                  NaN   \n",
       "186                                                NaN                  NaN   \n",
       "187                                                NaN                  NaN   \n",
       "188                                                NaN                  NaN   \n",
       "189                                                NaN                  NaN   \n",
       "190                                                NaN                  NaN   \n",
       "191                                                NaN                  NaN   \n",
       "192                                                NaN                  NaN   \n",
       "193                                                NaN                  NaN   \n",
       "194                                                NaN                  NaN   \n",
       "195                                                NaN                  NaN   \n",
       "196                                                NaN                  NaN   \n",
       "197                                                NaN                  NaN   \n",
       "198                                                NaN                  NaN   \n",
       "199                                                NaN                  NaN   \n",
       "\n",
       "    submission_alias submission_accession  \\\n",
       "0     GEO: GSE116170            SRA726535   \n",
       "1     GEO: GSE116170            SRA726535   \n",
       "2     GEO: GSE116170            SRA726535   \n",
       "3     GEO: GSE116170            SRA726535   \n",
       "4     GEO: GSE116170            SRA726535   \n",
       "5     GEO: GSE115607            SRA719221   \n",
       "6     GEO: GSE115607            SRA719221   \n",
       "7     GEO: GSE115607            SRA719221   \n",
       "8     GEO: GSE115607            SRA719221   \n",
       "9     GEO: GSE115607            SRA719221   \n",
       "10    GEO: GSE115607            SRA719221   \n",
       "11    GEO: GSE115607            SRA719221   \n",
       "12    GEO: GSE115607            SRA719221   \n",
       "13    GEO: GSE115607            SRA719221   \n",
       "14    GEO: GSE115607            SRA719221   \n",
       "15    GEO: GSE109891            SRA653941   \n",
       "16    GEO: GSE109891            SRA653941   \n",
       "17    GEO: GSE109891            SRA653941   \n",
       "18    GEO: GSE109891            SRA653941   \n",
       "19    GEO: GSE109891            SRA653941   \n",
       "20    GEO: GSE109891            SRA653941   \n",
       "21    GEO: GSE109891            SRA653941   \n",
       "22    GEO: GSE109891            SRA653941   \n",
       "23    GEO: GSE109891            SRA653941   \n",
       "24    GEO: GSE109891            SRA653941   \n",
       "25    GEO: GSE109891            SRA653941   \n",
       "26    GEO: GSE103023            SRA602384   \n",
       "27    GEO: GSE103023            SRA602384   \n",
       "28    GEO: GSE103023            SRA602384   \n",
       "29    GEO: GSE103023            SRA602384   \n",
       "..               ...                  ...   \n",
       "170    GEO: GSE94493            SRA535839   \n",
       "171    GEO: GSE94493            SRA535839   \n",
       "172    GEO: GSE94493            SRA535839   \n",
       "173    GEO: GSE94493            SRA535839   \n",
       "174    GEO: GSE94493            SRA535839   \n",
       "175    GEO: GSE94493            SRA535839   \n",
       "176    GEO: GSE94493            SRA535839   \n",
       "177    GEO: GSE94493            SRA535839   \n",
       "178    GEO: GSE94493            SRA535839   \n",
       "179    GEO: GSE94493            SRA535839   \n",
       "180    GEO: GSE94493            SRA535839   \n",
       "181    GEO: GSE94493            SRA535839   \n",
       "182              NaN                  NaN   \n",
       "183              NaN                  NaN   \n",
       "184              NaN                  NaN   \n",
       "185              NaN                  NaN   \n",
       "186              NaN                  NaN   \n",
       "187              NaN                  NaN   \n",
       "188              NaN                  NaN   \n",
       "189              NaN                  NaN   \n",
       "190              NaN                  NaN   \n",
       "191              NaN                  NaN   \n",
       "192              NaN                  NaN   \n",
       "193              NaN                  NaN   \n",
       "194              NaN                  NaN   \n",
       "195              NaN                  NaN   \n",
       "196              NaN                  NaN   \n",
       "197              NaN                  NaN   \n",
       "198              NaN                  NaN   \n",
       "199              NaN                  NaN   \n",
       "\n",
       "                                           study_title  \\\n",
       "0    Transcriptional properties of estrogen recepto...   \n",
       "1    Transcriptional properties of estrogen recepto...   \n",
       "2    Transcriptional properties of estrogen recepto...   \n",
       "3    Transcriptional properties of estrogen recepto...   \n",
       "4    Transcriptional properties of estrogen recepto...   \n",
       "5    ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "6    ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "7    ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "8    ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "9    ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "10   ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "11   ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "12   ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "13   ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "14   ChIP-Seq analysis of estrogen deprived MCF7 ce...   \n",
       "15   Clinical and genomic crosstalk between glucoco...   \n",
       "16   Clinical and genomic crosstalk between glucoco...   \n",
       "17   Clinical and genomic crosstalk between glucoco...   \n",
       "18   Clinical and genomic crosstalk between glucoco...   \n",
       "19   Clinical and genomic crosstalk between glucoco...   \n",
       "20   Clinical and genomic crosstalk between glucoco...   \n",
       "21   Clinical and genomic crosstalk between glucoco...   \n",
       "22   Clinical and genomic crosstalk between glucoco...   \n",
       "23   Clinical and genomic crosstalk between glucoco...   \n",
       "24   Clinical and genomic crosstalk between glucoco...   \n",
       "25   Clinical and genomic crosstalk between glucoco...   \n",
       "26   Estrogen receptor ChIP-seq in response to mTOR...   \n",
       "27   Estrogen receptor ChIP-seq in response to mTOR...   \n",
       "28   Estrogen receptor ChIP-seq in response to mTOR...   \n",
       "29   Estrogen receptor ChIP-seq in response to mTOR...   \n",
       "..                                                 ...   \n",
       "170  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "171  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "172  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "173  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "174  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "175  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "176  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "177  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "178  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "179  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "180  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "181  ChIPseq and RNAseq of ER mutant breast cancer ...   \n",
       "182                                                NaN   \n",
       "183                                                NaN   \n",
       "184                                                NaN   \n",
       "185                                                NaN   \n",
       "186                                                NaN   \n",
       "187                                                NaN   \n",
       "188                                                NaN   \n",
       "189                                                NaN   \n",
       "190                                                NaN   \n",
       "191                                                NaN   \n",
       "192                                                NaN   \n",
       "193                                                NaN   \n",
       "194                                                NaN   \n",
       "195                                                NaN   \n",
       "196                                                NaN   \n",
       "197                                                NaN   \n",
       "198                                                NaN   \n",
       "199                                                NaN   \n",
       "\n",
       "                            study_type  \\\n",
       "0    {'@existing_study_type': 'Other'}   \n",
       "1    {'@existing_study_type': 'Other'}   \n",
       "2    {'@existing_study_type': 'Other'}   \n",
       "3    {'@existing_study_type': 'Other'}   \n",
       "4    {'@existing_study_type': 'Other'}   \n",
       "5    {'@existing_study_type': 'Other'}   \n",
       "6    {'@existing_study_type': 'Other'}   \n",
       "7    {'@existing_study_type': 'Other'}   \n",
       "8    {'@existing_study_type': 'Other'}   \n",
       "9    {'@existing_study_type': 'Other'}   \n",
       "10   {'@existing_study_type': 'Other'}   \n",
       "11   {'@existing_study_type': 'Other'}   \n",
       "12   {'@existing_study_type': 'Other'}   \n",
       "13   {'@existing_study_type': 'Other'}   \n",
       "14   {'@existing_study_type': 'Other'}   \n",
       "15   {'@existing_study_type': 'Other'}   \n",
       "16   {'@existing_study_type': 'Other'}   \n",
       "17   {'@existing_study_type': 'Other'}   \n",
       "18   {'@existing_study_type': 'Other'}   \n",
       "19   {'@existing_study_type': 'Other'}   \n",
       "20   {'@existing_study_type': 'Other'}   \n",
       "21   {'@existing_study_type': 'Other'}   \n",
       "22   {'@existing_study_type': 'Other'}   \n",
       "23   {'@existing_study_type': 'Other'}   \n",
       "24   {'@existing_study_type': 'Other'}   \n",
       "25   {'@existing_study_type': 'Other'}   \n",
       "26   {'@existing_study_type': 'Other'}   \n",
       "27   {'@existing_study_type': 'Other'}   \n",
       "28   {'@existing_study_type': 'Other'}   \n",
       "29   {'@existing_study_type': 'Other'}   \n",
       "..                                 ...   \n",
       "170  {'@existing_study_type': 'Other'}   \n",
       "171  {'@existing_study_type': 'Other'}   \n",
       "172  {'@existing_study_type': 'Other'}   \n",
       "173  {'@existing_study_type': 'Other'}   \n",
       "174  {'@existing_study_type': 'Other'}   \n",
       "175  {'@existing_study_type': 'Other'}   \n",
       "176  {'@existing_study_type': 'Other'}   \n",
       "177  {'@existing_study_type': 'Other'}   \n",
       "178  {'@existing_study_type': 'Other'}   \n",
       "179  {'@existing_study_type': 'Other'}   \n",
       "180  {'@existing_study_type': 'Other'}   \n",
       "181  {'@existing_study_type': 'Other'}   \n",
       "182                                NaN   \n",
       "183                                NaN   \n",
       "184                                NaN   \n",
       "185                                NaN   \n",
       "186                                NaN   \n",
       "187                                NaN   \n",
       "188                                NaN   \n",
       "189                                NaN   \n",
       "190                                NaN   \n",
       "191                                NaN   \n",
       "192                                NaN   \n",
       "193                                NaN   \n",
       "194                                NaN   \n",
       "195                                NaN   \n",
       "196                                NaN   \n",
       "197                                NaN   \n",
       "198                                NaN   \n",
       "199                                NaN   \n",
       "\n",
       "                                        study_abstract  \n",
       "0    RNA sequencing (RNA-seq) detects estrogen rece...  \n",
       "1    RNA sequencing (RNA-seq) detects estrogen rece...  \n",
       "2    RNA sequencing (RNA-seq) detects estrogen rece...  \n",
       "3    RNA sequencing (RNA-seq) detects estrogen rece...  \n",
       "4    RNA sequencing (RNA-seq) detects estrogen rece...  \n",
       "5    The goal of this experiment was to interrogate...  \n",
       "6    The goal of this experiment was to interrogate...  \n",
       "7    The goal of this experiment was to interrogate...  \n",
       "8    The goal of this experiment was to interrogate...  \n",
       "9    The goal of this experiment was to interrogate...  \n",
       "10   The goal of this experiment was to interrogate...  \n",
       "11   The goal of this experiment was to interrogate...  \n",
       "12   The goal of this experiment was to interrogate...  \n",
       "13   The goal of this experiment was to interrogate...  \n",
       "14   The goal of this experiment was to interrogate...  \n",
       "15   Steroid hormone receptors are simultaneously a...  \n",
       "16   Steroid hormone receptors are simultaneously a...  \n",
       "17   Steroid hormone receptors are simultaneously a...  \n",
       "18   Steroid hormone receptors are simultaneously a...  \n",
       "19   Steroid hormone receptors are simultaneously a...  \n",
       "20   Steroid hormone receptors are simultaneously a...  \n",
       "21   Steroid hormone receptors are simultaneously a...  \n",
       "22   Steroid hormone receptors are simultaneously a...  \n",
       "23   Steroid hormone receptors are simultaneously a...  \n",
       "24   Steroid hormone receptors are simultaneously a...  \n",
       "25   Steroid hormone receptors are simultaneously a...  \n",
       "26   We have used ChIP-seq to profile binding of th...  \n",
       "27   We have used ChIP-seq to profile binding of th...  \n",
       "28   We have used ChIP-seq to profile binding of th...  \n",
       "29   We have used ChIP-seq to profile binding of th...  \n",
       "..                                                 ...  \n",
       "170  We performed several extensive phenotypic, gen...  \n",
       "171  We performed several extensive phenotypic, gen...  \n",
       "172  We performed several extensive phenotypic, gen...  \n",
       "173  We performed several extensive phenotypic, gen...  \n",
       "174  We performed several extensive phenotypic, gen...  \n",
       "175  We performed several extensive phenotypic, gen...  \n",
       "176  We performed several extensive phenotypic, gen...  \n",
       "177  We performed several extensive phenotypic, gen...  \n",
       "178  We performed several extensive phenotypic, gen...  \n",
       "179  We performed several extensive phenotypic, gen...  \n",
       "180  We performed several extensive phenotypic, gen...  \n",
       "181  We performed several extensive phenotypic, gen...  \n",
       "182                                                NaN  \n",
       "183                                                NaN  \n",
       "184                                                NaN  \n",
       "185                                                NaN  \n",
       "186                                                NaN  \n",
       "187                                                NaN  \n",
       "188                                                NaN  \n",
       "189                                                NaN  \n",
       "190                                                NaN  \n",
       "191                                                NaN  \n",
       "192                                                NaN  \n",
       "193                                                NaN  \n",
       "194                                                NaN  \n",
       "195                                                NaN  \n",
       "196                                                NaN  \n",
       "197                                                NaN  \n",
       "198                                                NaN  \n",
       "199                                                NaN  \n",
       "\n",
       "[200 rows x 14 columns]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "from IPython.display import display, HTML\n",
    "display(df)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
